### ISSUE TITLE
DateRangeInput placeholder not working

### ISSUE THREAD

**Issue by dessskris** [palantir/blueprint#4487]
#### Environment

- __Package version(s)__: @blueprintjs/datetime 3.18.5
- __Operating System__: macOS Big Sur Version 11.1
- __Browser name and version__: Chrome Version 87.0.4280.141 (Official Build) (x86_64)

#### Code Sandbox

Link to a minimal repro: https://codesandbox.io/s/blueprint-sandbox-forked-rk722?file=/src/DatetimeExample.tsx

#### Steps to reproduce

Specify `placeholder` on `DateRangeInput` e.g. "Please enter date"

#### Actual behavior

Placeholders say "Start date" and "End date"

#### Expected behavior

Placeholders should be the specified string e.g. "Please enter date"

#### Possible solution

Since there are two fields, the placeholder should be `[string, string]` for the start date and end date respectively

**Comment by adidahiya** [comment]
Good catch. Looks like we haven't added proper support for that prop even though it appears in the props interface. Open to PRs to fix this.
